packages/analytics/analytics-utilities/src/types/analytics-data.ts

```diff
@@ -18,6 +18,10 @@ export type MetricUnit = {
   [metricName in ExploreAggregations]?: string
 }
 
+export type AiMetricUnit = {
```

Comment:
I think there should be a type that represents a superset of all possible aggregations.  Something like AllAggregations = ExploreAggregations | AiExploreAggregations.  Then drive MetricUnit off of that (instead of adding AiMetricUnit, and then type metric_names and metric_units off of the AllAggregations type.

Logically, I think that makes the most sense: AnalyticsChart needs to be able to handle all possible aggregations -- not just explore aggregations.